### ISSUE TITLE
Get "NaN" loss when training Libra R-CNN on our own dataset

### ISSUE THREAD

**Issue by qWildManp** [open-mmlab/mmdetection#3551]
**Notice**
2. Reimplement a model in the model zoo on other dataset (e.g., custom datasets)

**Describe the issue**
We are trying to reimplement the Libra R-CNN model on our own dataset. The dataset is modified into VOC format. We run the model on Google Colab and we strictly follow the  instruction to set up mmdetection enviornment.The LIbra R-CNN model works fine when training on VOC dataset. Howerver,when training on our own dataset we found that when the model is running in around 400 to 500 iterations or the loss is around 0.4, the loss would suddenly become NaN.
We tried many methods to sovle it, including lower and larger the learning rate and batch size, add gradient clipping,but these methods doesn't work. We are confused  how this would happened
**Reproduction**
1. What command or script did you run?
```
!python tools/train.py ./configs/libra_rcnn/libra_faster_rcnn_x101_64x4d_fpn_1x_voc.py
```
2. What config dir you run?
```
 ./configs/libra_rcnn/libra_faster_rcnn_x101_64x4d_fpn_1x_voc.py
```
3. Did you make any modifications on the code or config? Did you understand what you have modified?
We only change the config of dataset from COCO to VOC
4. What dataset did you use?
Our own dataset in VOC format
**Environment**

1. Please run `python mmdet/utils/collect_env.py` to collect necessary environment information and paste it here.
sys.platform: linux
Python: 3.6.9 (default, Jul 17 2020, 12:50:27) [GCC 8.4.0]
CUDA available: True
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 10.1, V10.1.243
GPU 0: Tesla P100-PCIE-16GB
GCC: gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
PyTorch: 1.5.1+cu101
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2019.0.5 Product Build 20190808 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v0.21.1 (Git Hash 7d2fd500bc78936d1d648ca713b901012f470dbc)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 10.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_37,code=compute_37
  - CuDNN 7.6.3
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_INTERNAL_THREADPOOL_IMPL -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.6.1+cu101
OpenCV: 4.1.2
MMCV: 1.0.5
MMDetection: 2.3.0+b97322e
MMDetection Compiler: GCC 7.5
MMDetection CUDA Compiler: 10.1
2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]
using conda install

**Comment by OceanPang** [comment]
Hi @qWildManp , sorry that I can not directly debug for your issue as the short of your datasets. 
Can you first try to give a longer warming up schedule? The default setting on VOC does not have warm-up but we do have it on other larger and more challenge datasets. 
If this still do not work, please try to use Faster R-CNN first. If it works, try to add the component of Libra R-CNN one by one. We may find the issue here in this way.
